Documentation/how-to-build-and-run-ilcompiler-in-vscode.md

```diff
@@ -0,0 +1,91 @@
+_Please ensure that [pre-requisites](prerequisites-for-building.md) are installed for a successful build of the repo._
+
+_Note_:
+
+* Instructions below assume `~/corert` is the repo root.
+
+# Setting up #
+
+Please make sure you have latest VS Code, C# extension, and .NET Core available. This guide is tested under C# 1.6.2 + VS Code 1.8.1 + CLI 1.0.0-preview4-004233.
+
+This guide assumes that your VS code workspace is set to the root of the repo. 
+
+# Enabling Building from VS Code #
+
+Press ```SHIFT+COMMAND+B```. VS code will automatically ask you to configure the Task Runner. Choose 'Other', then change your tasks.json as follows:
+
+* Change command to ```${workspaceRoot}/build.sh```
+* Add ```"suppressTaskName" : true``` under the build Task. This is important to avoid always passing "build" task name as argument to build.sh. 
+
+The full tasks.json is as follows:
```

Comment:
Is it possible to checkin these files so that you do not have to hand-create them?